akka-docs/src/main/paradox/stream/stream-graphs.md

```diff
@@ -86,7 +86,18 @@ Scala
 :   @@snip [GraphDSLDocSpec.scala]($code$/scala/docs/stream/GraphDSLDocSpec.scala) { #graph-dsl-reusing-a-flow }
 
 Java
-:   @@snip [GraphDSLDocTest.java]($code$/java/jdocs/stream/GraphDSLDocTest.java) { #graph-dsl-reusing-a-flow }
+:   @@snip [GraphDSLTest.java]($akka$/akka-stream-tests/src/test/java/akka/stream/javadsl/GraphDslTest.java) { #graph-dsl-reusing-a-flow }
+
+Some extreme cases we have a (generated) list of graph elements. 
+For example we dynamically generate some filters, and want to collect the results to multiple sources (maybe writing different files simultaneously).
+If these graph elements have similar signatures, there is a possibility to construct a graph from them.
```

Comment:
Minor language fixes, something like:

"In some cases we may have a list of graph elements, for example if they are dynamically created. If these graphs have similar signatures, we can construct a graph collecting all their materialized values as  a collection:"